<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="chartContainer" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>681</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>451</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string>Статистика</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Статистические методы обработки данных</string>
        </property>
        <item>
         <property name="text">
          <string>Параметрические методы</string>
         </property>
         <item>
          <property name="text">
           <string>Метод максимального правдоподобия</string>
          </property>
          <item>
           <property name="text">
            <string>Нормальное распределение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Распределение Вейбулла-Гнеденко</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Метод наименьших квадратов</string>
          </property>
          <item>
           <property name="text">
            <string>Нормальное распределение MLS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Распределение Вейбулла-Гнеденко MLS</string>
           </property>
          </item>
         </item>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Статистические критерии обработки данных</string>
        </property>
        <item>
         <property name="text">
          <string>Параметрические критерии</string>
         </property>
         <item>
          <property name="text">
           <string>Критерий Граббса</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Критерий Фишера-Стьюдента</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Однофакторный дисперсионный анализ (ANOVA)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Критерий Шапило-Уилка (W-критерий)</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Непараметрические критерии</string>
         </property>
         <item>
          <property name="text">
           <string>Двухвыборочный критерий Уилкоксона</string>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_output">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>273</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>760</y>
      <width>451</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_1">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_load">
       <property name="text">
        <string>Загрузить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_2">
       <property name="text">
        <string>Сохранить данные</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
